docs/how_to_dev_backend_en.md

```diff
@@ -0,0 +1,124 @@
+# VisualDL Backend / SDK structure and Develop Guide
+
+VisualDL has three components.
+1. The Python/C++ SDK that logs the data during training.
+1. The single page client side app that visualized training data.
+1. The server (powered on Flask) that reads the logs data and delivers it to the client side app for displaying graphs (scalars/histograms) and embeddings.
+
+This doc will go over Backend / SDK structure with development guide.
+
+For front end architecture and development guide, looks [here](./how_to_dev_frontend_en.md)
+
+
+## Code structure
+
+All backend and sdk logic is under visualdl sub directory
+
+```
+├── build
+├── build.sh
+├── CMakeLists.txt
+├── frontend
+├── visualdl
+    ├── logic    - API for C++ SDK, information maintainer 
+    ├── python   - API for Python SDK
+    ├── server   - Flask server, provides front end APIs for read data
+    ├── storage  - Data structure and protobuf
+    └── utils    - Helper function and classes
+```
+
+## Development
+
+Any code changes in ```server``` folder, simply run
+
+```
+python visualdl/server/visualDL --logdir={LOG_DIR} --port=8080
```

Comment:
So I had a problem on this step wherein it was importing libraries from the pip installed version of `visualDL`. So I had to resolve it by resetting my PYTHONPATH. You don't need to explain it as much as it would be nice if it is mentioned in a comment.

Reply:
ya, I run into same issue, let me update